gym/spaces/tuple.py

```diff
@@ -16,7 +17,9 @@ class Tuple(Space[tuple], Sequence):
     """
 
     def __init__(
-        self, spaces: Iterable[Space], seed: Optional[Union[int, List[int]]] = None
+        self,
+        spaces: Iterable[Space],
+        seed: Optional[Union[int, seeding.RandomNumberGenerator]] = None,
```

Comment:
Why remove the option of passing a sequence of ints?
```suggestion
        seed: Optional[Union[int, Sequence[int], seeding.RandomNumberGenerator]] = None,
```